alert-common/src/main/java/com/synopsys/integration/alert/common/descriptor/config/field/endpoint/table/EndpointTableSelectField.java

```diff
@@ -25,17 +25,17 @@
 import java.util.LinkedList;
 import java.util.List;
 
-import com.synopsys.integration.alert.common.action.CustomFunctionAction;
 import com.synopsys.integration.alert.common.descriptor.config.field.endpoint.EndpointField;
 import com.synopsys.integration.alert.common.enumeration.FieldType;
+import com.synopsys.integration.alert.common.rest.api.AbstractFunctionController;
 
 public class EndpointTableSelectField extends EndpointField {
     private boolean paged;
     private boolean searchable;
-    private List<TableSelectColumn> columns;
+    private final List<TableSelectColumn> columns;
```

Comment:
why the use of final here?

Reply:
Intellij must have added it. Good catch, I'll look into it.

Reply:
Looks like it's because the only operations on `columns` are additive.